go.mod

```diff
@@ -1,14 +1,23 @@
 module github.com/arianvp/webauthn-oidc
 
-go 1.16
+go 1.18
 
 replace github.com/duo-labs/webauthn => github.com/arianvp/webauthn v0.0.0-20210928001254-d9dcd044e9f9
 
 require (
 	github.com/duo-labs/webauthn v0.0.0-20210727191636-9f1b88ef44cc
-	github.com/google/go-cmp v0.5.6 // indirect
 	github.com/gorilla/sessions v1.2.1
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.7.0 // indirect
-	gopkg.in/square/go-jose.v2 v2.6.0
+)
+
+require (
+	github.com/cloudflare/cfssl v0.0.0-20190726000631-633726f6bcb7 // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
```

Comment:
It might be a good idea to look at where some of these packages are coming from and how they’re being used. Particularly because this specific package has been archived, _and_ has an open security issue.

Reply:
They're pulled in by `duo-labs/webauthn` which I'm replacing with `arianvp/webauthn-minimal` in a follow-up PR